src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -1063,8 +1061,12 @@ function SetupElements(container) {
         }
     });
 
-    //focus on the first editable field
-    $(container).find("textarea, div.bloom-editable").first().focus(); //review: this might choose a textarea which appears after the div. Could we sort on the tab order?
+    // focus on the first editable field
+    // HACK for BL-1139: except for some reason when the Reader tools are active this causes
+    // quick typing on a newly loaded page to get the cursor messed up. So for the Reader tools, the
+    // user will need to actually click in the div to start typing.
+    if (!model)
+        $(container).find("textarea, div.bloom-editable").first().focus(); //review: this might choose a textarea which appears after the div. Could we sort on the tab order?
```

Comment:
I don't think it's worth redoing, but for future reference, I'd prefer not to spread around knowledge like "the way to know whether the reader tools are loaded is to check whether model isn't null". Even though it looks trivial, creating a function readerToolsLoaded() {return model;} documents what you are doing and makes it easy to change if at some point we have a model even when they aren't loaded, or stop having direct access to the model object.